independent-publisher-connectors/Pushover/apiDefinition.swagger.json

```diff
@@ -0,0 +1,361 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Pushover",
+    "description": "Pushover makes it easy to get real-time notifications on your Android, iPhone, iPad, and Desktop.",
+    "contact": {
+      "name": "Glen Hutson",
+      "email": "msconnectors@glenhutson.com"
```

Comment:
@glenhutson is possible to add a "url" to the contact section. this could be a GitHub page or LinkedIn for example.

Reply:
@Amjed-Ayoub, URL added.